03-components/50-weather-components/WeatherApp.js

```diff
@@ -1,52 +1,31 @@
-import { defineComponent } from 'vue'
+import { defineComponent, ref } from 'vue'
 import { getWeatherData, WeatherConditionIcons } from './weather.service.ts'
+import { UiContainer } from '@shgk/vue-course-ui';
+import WeatherList from './WeatherList.js';
 import './WeatherApp.css'
 
 export default defineComponent({
   name: 'WeatherApp',
 
+  components: {
+    UiContainer,
+    WeatherList
+  },
+
+  setup() {
+    const weatherData = ref(getWeatherData());
+    const weatherConditionIcons = ref(WeatherConditionIcons);
```

Comment:
Same, `ref` is not needed here, until the values is changing during component instance life, and there is a need to react on it.